profile/README.md

```diff
@@ -25,7 +27,18 @@ Exceptionally talented open communities are building the future of the decentral
 
 ### Implementations
 
-- [Kubo (Go)](https://github.com/ipfs/kubo)
+- [Kubo (Go)](https://github.com/ipfs/kubo) + ([Cluster](https://github.com/ipfs-cluster/ipfs-cluster) for orchestration)
+- [Iroh (Rust)](https://github.com/n0-computer/iroh)
 - [Helia (JS)](https://github.com/ipfs/helia)
 - and [more!](https://docs.ipfs.tech/concepts/ipfs-implementations/#popular-or-actively-maintained)
 
+### Integrations, Tooling, & Plugins
```

Comment:
What do you think about adding this section instead to https://github.com/ipfs/awesome-ipfs and just linking to the list from here?